src/meta/test/JobManagerTest.cpp

```diff
@@ -170,6 +171,7 @@ TEST_F(JobManagerTest, IngestJob) {
   ASSERT_EQ(code, nebula::cpp2::ErrorCode::SUCCEEDED);
   code = executor->execute();
   ASSERT_EQ(code, nebula::cpp2::ErrorCode::SUCCEEDED);
+  delete executor;
```

Comment:
Better to use `make_unique<IngestJobExecutor>(space, job.getJobId(), kv.get(), &adminClient, job.getParas())` to avoid explicit `new/delete`?